src/main/java/nextstep/ladder/domain/Direction.java

```diff
@@ -1,14 +1,63 @@
 package nextstep.ladder.domain;
 
+import nextstep.ladder.util.RandomUtil;
+
+import java.util.Objects;
+
 public class Direction {
-    private int direction;
+    private boolean current;
+    private boolean next;
```

Comment:
```suggestion
    private final boolean current;
    private final boolean next;
```
전체적으로 변경되지 않는 값에 대해 final을 붙여주세요 :)